docs/source/_toctree.yml

```diff
@@ -0,0 +1,21 @@
+- sections:
+  - local: index
+    title: Text Generation Inference
+  - local: quicktour
+    title: Quick Tour
+  - local: supported_models
+    title: Supported Models and Hardware
+  title: Getting started
+- sections:
+  - local: basic_tutorials/local_launch
+    title: Installing and Launching Locally
```

Comment:
I think the installation would make more sense in the "Getting started" section

Reply:
Should we have Installation & Launch both in Getting Started?

Reply:
Yeah I think those are good first topics to cover; users should have TGI set up first so they're ready to start consuming the tutorials!

Reply:
I think it's Docker.

Reply:
Maybe then have a clear message at the top of this section that this is only needed if you want to build things from scratch.

The section name also sounds off. This is not about launching locally (the docker instructions do that too), this is about installing from source.

Reply:
Addressed this.